modules/swagger-codegen/src/main/java/io/swagger/codegen/CodegenOperation.java

```diff
@@ -19,7 +19,7 @@
             isRestfulIndex, isRestfulShow, isRestfulCreate, isRestfulUpdate, isRestfulDestroy,
             isRestful, isDeprecated;
     public String path, operationId, returnType, httpMethod, returnBaseType,
-            returnContainer, summary, unescapedNotes, notes, baseName, defaultResponse, discriminator;
+            returnContainer, summary, unescapedNotes, notes, baseName, defaultResponse, discriminator, description;
```

Comment:
@ToreJohansson I think "notes" is the operation's description. Have you tried it?

Reply:
In here it is defined as "description":
https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md

I am running this patch. It works well. Good to get the description field as java-doc in the generated code.

Reply:
@ToreJohansson I mean did you try {{notes}}? I agree it should be named "description" instead but it was named "notes" based on historical reason.